Library/Formula/dnscrypt-proxy.rb

```diff
@@ -39,6 +40,32 @@ def install
 
     system "./configure", *args
     system "make", "install"
+
+    (bin/"dnscrypt-update-resolvers").write <<-EOS.undent
```

Comment:
Sorry, I was unclear. This should still be wrapped in a `if build.with? "minisign"` block.